atomic

```diff
@@ -273,6 +280,9 @@ if __name__ == '__main__':
     imagesp.set_defaults(func='images')
     imagesp.add_argument("--prune", action="store_true",
                          help=_("prune dangling images"))
+    imagesp.add_argument("--oci", dest="oci",
+                         action='store_true', default=False,
+                         help=_('Show OCI containers'))
 
```

Comment:
I like this idea, just the output has a different format (as most of the Docker fields have no equivalent in an OSTree repository).  Is there anything depending on the format of the `atomic images` output?  How will we print both formats?

This is the output I have right now:

```
bash-4.3# atomic images
REPOSITORY                          TAG    IMAGE ID     CREATED             VIRTUAL SIZE
 docker.io/gscrivano/spc-helloworld latest sha256:aa510 2016-03-22 12:23    216.41 MB 
 docker.io/gscrivano/spc-flannel    latest sha256:9b5dd 2016-03-21 13:44    230.42 MB 
 docker.io/busybox                  latest sha256:47bcc 2016-03-18 18:22    1.11 MB   
 docker.io/alpine                   latest sha256:70c55 2016-03-02 17:16    4.79 MB   
bash-4.3# atomic images --oci
IMAGE                                    COMMIT                                                          
dockerimg/busybox-latest                 ff6b0e9239c9569be3ef48699519d7842975a48c98b0ba560000363006364c5e
dockerimg/gscrivano/spc-flannel-latest   00b316952aee741a2eeb62c98e7dc7a2018c6ce94b1e3cd93ba0ce12955c3cf3
```